src/components/CurrencyField/CurrencyField.scss

```diff
@@ -3,7 +3,7 @@
 
 .field-container {
   background: $white;
-  border: 1px solid $black-300;
+  border: 1.5px solid $black-300;
```

Comment:
I think fractional pixels are truncated

Reply:
I thought so too, but it actually does work and it makes a difference (this value was specifically requested by the design team)